client/go/util/http.go

```diff
@@ -38,20 +29,6 @@ func (c *defaultHttpClient) UseCertificate(certificates []tls.Certificate) {
 	}}
 }
 
-func CreateClient(timeout time.Duration) HttpClient {
-	return &defaultHttpClient{
-		client: &http.Client{Timeout: timeout},
-	}
-}
-
-func HttpDo(request *http.Request, timeout time.Duration, description string) (*http.Response, error) {
-	if request.Header == nil {
-		request.Header = make(http.Header)
-	}
-	request.Header.Set("User-Agent", fmt.Sprintf("Vespa CLI/%s", build.Version))
```

Comment:
Looks like this entirely removed the User-Agent override?
I think i would be best to keep that, then we can keep somewhat track of older versions in use.